src/screens/OtherProfile/elements/BioAndChat.js

```diff
@@ -148,7 +148,7 @@ const BioAndChat = (props) => {
           labelLeft={
             isAnonimityEnabled || !isSignedMessageEnabled ? 'Incognito' : 'Incognito disabled'
           }
-          styleLabelLeft={{color: COLORS.white}}
+          styleLabelLeft={{color: COLORS.gray500}}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.gray500` for the styleLabelLeft color might affect readability depending on the background color. Please ensure that this color change is still accessible and provides enough contrast for users.
```diff
- styleLabelLeft={{color: COLORS.white}}
+ styleLabelLeft={{color: COLORS.gray500}}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->